css/dark-theme.css

```diff
@@ -69,27 +73,47 @@ html.dark-mode #logo > a {
   color: var(--dark_main_text);
 }
 /* navbar links/drop down menu links */
-html.dark-mode #navbar ul#navmenu li a,
-html.dark-mode #navbar ul#navmenu li.dropit-trigger a {
+html.dark-mode #navbar ul#navmenu li a {
   color: var(--dark_main_text);
 }
 /* first drop down link - some js is adding current class */
 html.dark-mode #navbar ul#navmenu li a.current {
   background: transparent;
 }
-html.dark-mode #navbar ul#navmenu li a.current:hover {
+html.dark-mode #navbar ul#navmenu li a.current:hover,
+html.dark-mode .submenu-content a:hover {
   background: var(--dark_hover);
 }
-html.dark-mode .menu ul.dropit-submenu {
-  background: var(--dark_bg);
+
+html.dark-mode #navmenu > li {
+  background: var(--second_dark_bg);
 }
-/* drop down menu links  */
-html.dark-mode #navbar .menu ul.dropit-submenu a:hover {
-  background: var(--dark_hover);
+
+html.dark-mode #q {
+  border-color: #979797 !important;
```

Comment:
**We don't want to use important.** 
Make the element more specific using `div#q`, something like this might work
`html.dark-mode section#logo ~ div#navbar  input#q`

I remember using it here b/c someone else used it in style.css so I had to use to overide that. We only want important as a last resort. 

If it wasn't used in the style.css already, I'd say we should NEVER use it.

Reply:
done